tests/functional/tests/bundles/base-yoga-ovn.yaml

```diff
@@ -169,6 +177,9 @@ relations:
   - - 'nova-compute:amqp'
     - 'rabbitmq-server:amqp'
 
+  - - nova-compute:secrets-storage
+    - vault:secrets
```

Comment:
If nova-compute worked without vault previously, not sure why we need it now? I think it's because of the `encrypt: true` option to nova-compute. Do we need encryption there? Can we remove that option?